README.md

```diff
@@ -44,8 +45,32 @@ Edit the `local.json` before you start the build. set the right
 `iso_url` and the right `iso_checksum_url`. Then you can start the build
 for virtualbox only with the following command:
 
+```bash
+jq --join-output '.["post-processors"][0] |= map(select(.type == "vagrant"))' vagrant.json \
+| tr -d '\r' \
+> vagrant_local.json \
+&& packer build -only=virtualbox-iso -var-file=local.json vagrant_local.json \
+&& rm vagrant_local.json
+```
+
+In [Cmder](https://cmder.net/) on Windows, which is persnickety with quotation marks, that command would be the following:
```

Comment:
Why not using `WSL` (Windows Subsystem for Linux)? I think we should stick with the Linux way and Windows users can just use the `WSL` to preprocess the json file